### DISCUSSION TITLE
[Q&A] 能否出一个lobechat 和自定义插件业务逻辑的说明或文档

### DISCUSSION THREAD

**Discussion by scortLi** [lobehub/lobehub#1057] [Development Question] [answered]
### 🥰 需求描述

建议能否出一个插件和lobechat如何调用的说明，比如什么条件下才会调用插件，是怎样判断的，还有插件是如何和gpt交互的这些逻辑，麻烦了

### 🧐 解决方案

我需要自定义插件但是不明白具体的实现逻辑

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
请查阅此篇： https://chat-plugin-sdk.lobehub.com/zh-CN/guides/plugin-invoke

[a comment by an automated account is not shown]
